### PR TITLE
[OTTO-346] Test on Drupal 9.

### PR DESCRIPTION
**Pull request by greg-1-anderson**
Note that in order to support Drupal 9, we must start using the Core Composer Scaffold tool, which means we need to drop support for PHP 5.x.

We will therefore make a new 2.x series, and continue to install the existing 1.x version on php 5.

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.1.10-dev
+2.0.0-dev
```
Comment (greg-1-anderson) [inline VERSION]: Don't need to bump major version

File: .gitattributes [hunk 2]
```diff
@@ -1,6 +1,61 @@
-/.gitattributes         export-ignore
-/.scenarios.lock        export-ignore
-/.travis.yml            export-ignore
-/sut                    export-ignore
-/tests                  export-ignore
-/phpunit.xml.dist       export-ignore
+# Drupal git normalization
```
Comment (greg-1-anderson) [inline .gitattributes]: Committed on accident

**Comment by greg-1-anderson** [comment]
Ignore everything in .scenarios.lock

File: src/Check/ExtensionsDuplicate.php [hunk 3]
```diff
@@ -86,7 +86,7 @@ public function getResultWarn() {
     else {
       foreach ($this->registry->extensions_dupe as $name => $extension_infos) {
         $ret_val .= PHP_EOL;
-        if (!drush_get_option('json')) {
+        if (true /* !drush_get_option('json') */) {
```
Comment (pwtyler) [inline src/Check/ExtensionsDuplicate.php:91]: Is this debugging that should be removed, or should the if statement just be taken out period?

**Review by pwtyler — CHANGES_REQUESTED** [review]
Looks like one debugging step was left in, or the if statement is no longer relevant and should be removed.

File: src/Check/ExtensionsDuplicate.php (hunk 3 above)
Comment (greg-1-anderson) [reply src/Check/ExtensionsDuplicate.php:91]: This is code that was never ported for Drush 9+

The old code calls `drush_get_option`, which is always a no-op in Drush 9, and is a "function not found" in Drush 10.

These classes have no access to the commandline options in local variables or fields; the original code relied on global state to optimize the output formatting in json mode. The work to refactor to get a few spaces out of the output is still tbd.

A better comment here would be helpful.

**Review by pwtyler — APPROVED** [review]

**Pull request merged**